README.md

```diff
@@ -1,16 +1,24 @@
 # kubevirt-image-service ![kubevirt-image-service](https://github.com/tmax-cloud/kubevirt-image-service/workflows/kubevirt-image-service/badge.svg)
-## kubevirt-image-service란?
-kubevirt-image-service(이하 kis)는 kubevirt에 통합되는 이미지, 볼륨 서비스를 제공합니다. (TODO: 데모)
 
-- `VirtualMachineImage`  : HTTP, S3, Local을 통해 외부에서 이미지를 임포트할 수 있습니다. (현재는 HTTP만 지원) 한 번 임포트된 이미지는 외부에서 다시 클론되지 않으며 내부에서 자체적으로 관리됩니다.
-- `VirtualMachineVolume` : 볼륨은 이미지로부터 생성할 수 있으며 CSI(Container Storage Interface)의 스냅샷-클론 기능을 활용하여 용량을 효율적으로 관리합니다.
-- `VirtualMachineExport` : 사용하던 볼륨을 qcow2 파일로 변환하여 외부에 전달할 수 있습니다.
+## What is kubevirt-image-service?
 
-## 시작하기
-다음 [유저 가이드](docs/USERGUIDE.md)를 참조하세요.
+kubevirt-image-service (KIS) is an integration with KubeVirt to provide image and volume services.  
```

Comment:
please specify the link to the CDI project and add why we started this project and what's diff with CDI

Reply:
Done.